user/languages/javascript-with-nodejs.md

```diff
@@ -121,6 +121,51 @@ npm install
 
 Note that `npm install` can fail if a shrinkwrapped git dependency pointing to a branch has its HEAD changed.
 
+## Ember Apps
+You can build your Ember Apps on Travis CI and can automate test. By default ember comes with [`Qunit`](http://qunitjs.com/) as testing framework
+
+```yaml
+dist: trusty
```

Comment:
can you add a `sudo: required` line right above `dist: trusty` ? That's a recommended practice per https://docs.travis-ci.com/user/trusty-ci-environment to route builds to Trusty :)